en/includes/guides/authentication/app-native-authentication/handle-advanced-login-scenarios.md

```diff
@@ -29,19 +32,34 @@ To do so,
 
 ### Redirect mode
 
-In the rediect mode, the application redirects the user to the IdP for authentication. Once it's complete, the application receives an authorization code from the IdP which is then sent back to {{product_name}}. ({{product_name}} users this to make a token call to the IdP.).
+In the rediect mode, the application redirects the user to the IdP as it does in a [conventional federated login flow]({{base_path}}/guides/authentication/federated-login/). However, under the hood, app-native authentication handles federated authentication slightly differently.
+
+!!! note "How is it different?"
+    
+    Although federated login flows in both [conventional login]({{base_path}}/guides/authentication/federated-login/) and app-native authentication redirect the user to the external IdP, there is a subtle difference between them.
+
+    - In a conventional flow, {{product_name}} redirects the user to the external IdP. Once the authentication is complete, the user is redirected back to {{product_name}} with an authorization code.
+
+    - In app-native authentication, {{product_name}} constructs a redirection URL (in which the callback URL is set to the application) and sends it to the application. The application redirects the user to the external IdP using the redirection URL. Once the authentication is complete, the IdP redirects the user back to the application (using the callback URL) with an authorization code. The application then sends the authorization code to {{product_name}}.
```

Comment:
Shall we add a bit more context here like `... in which the callback URL is set to the application using the callback URL sent in the initial authorization request)` IMO better to have this information so the developer will know exactly where the callback will be and be able to handle it in the app

Reply:
Ack. Please review the latest changes.

Reply:
![image](https://github.com/wso2/docs-is/assets/31583807/05138032-5f5a-43b2-98a7-baae70daf957)

Reply:
![image](https://github.com/wso2/docs-is/assets/31583807/b1777e81-d9e3-4cf9-9638-99a2818a8d3f)